docs/BasicCommunication/GetSystemTime/index.md

```diff
@@ -0,0 +1,109 @@
+## GeSystemTime
+
+Type
+: Function
+
+Sender
+: SDL
+
+Purpose
+: Obtain current UTC time  
+
+When an application  wants to open protected service it sends certificate to SDL during TLS or DTLS handshake.  
```

Comment:
When an application  wants to open protected service it sends certificate to SDL during TLS or DTLS handshake.  
should be
When an application needs to start a protected service with SDL core the app sends a certificate to SDL during the TLS or DTLS handshake process

Reply:
@mrapitis, thank you.
fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/94/commits/3c7a300842699307e9fd06ffb64fcd65d66d6d77